paper/paper.md

```diff
@@ -0,0 +1,105 @@
+bibliography: paper.bib
+---
+
+# Summary
+
+Game Theory and Python is an open source educational tutorial aimed to
+introduce readers to game theory and more specifically to repeated games. The
+tutorial uses the open source library called
+[Axelrod-Python](https://github.com/Axelrod-Python/Axelrod) and [Jupyter
+Notebooks](https://jupyter.org) making this an open source, reproducible and
+interactive tutorial.
+
+# Statement of Need
+
+Game theory is a field of applied mathematics and an exceptional tool for
+introducing students to the field. Students can start off by playing games, then
+conceptualizing strategies, and before long using equations to formulate the
+interactions between players.
+
+Repeated games are a set of games that are particularly popular in educational
+game theory, and a specific game that has gained much attention in both research
+(Axelrod1981) and teaching (reference) is the Prisoner's Dilemma. The advantage
+of studying the Prisoner's Dilemma is that it models situations in which
+self-interest clashes with collective interest, thus it provides a powerful
+framework for illustrating the usage of mathematics in real life decision
+making.
+
+% TODO a good reference that the PD is heavily used in education.
+
+# Content
+
+The four notebooks of this tutorial focus on the iterated version of the
+Prisoner's Dilemma, the Iterated Prisoner's Dilemma, the notion of strategies
+and computer tournaments whilst using Axelrod-Python.
+
+In 1980, a political scientist called Robert Axelrod decided to run a computer
+tournament of the Iterated Prisoner’s Dilemma where strategies written in
+computer code would repeatedly choose between self and collective interest. The
+strategies would decide on their next action using the history of previous
+interactions and the winner was decided on the average score. The open source
+package Axelrod-Python was created in order to reproduce Axelrod's original work
+but to also serve as an educational and research tool.
+
+Following the tutorial, participants will be able to reproduce Axelrod's
+tournament but also create their own unique tournaments from a selection of
+strategies, exceeding 200. Furthermore, participants will be able to progress
+their comprehension of the topic by producing strategies and implement them
+using Python code. The new strategies then can be placed in different
+tournaments in order to access their success and weakness. This progression
+of the tutorial is based on constructive framework. Participants
+start by following instructions given to them, followed by the participants
+producing their own strategies and tournament. This will allow them to fully
+the consequences of their decision making in a setting where they interact
+with others.
+
+%TODO references
+
+# Usage
+
+The tutorial can be used in any of the two ways; in a workshop environment or
+at home.
+
+- In a workshop environment we suggest that the instructor has familiarized
```

Comment:
@Nikoleta-v3 I think you should add a usage guide to your README:

Write out the suggested number of hours for the workshop and then write a loose timetable. 

Then you essentially repeat that here and also explicitly say that it's in the README. (You should also have a section for individual learners.)